spec/support/vcr.rb

```diff
@@ -38,11 +38,21 @@
     classroom_owner_organization_github_login
   end
 
+  # Owner token
+  c.filter_sensitive_data("<TEST_CLASSROOM_OWNER_GITHUB_TOKEN>") do
```

Comment:
We weren't calling the actual methods for filtering tokens in vcr, might want to refactor this file in the future so there doesn't have to be multiple calls.